examples/java/src/main/java/org/apache/beam/examples/cookbook/README.md

```diff
@@ -39,7 +39,7 @@ larger Dataflow pipeline. They include:
   </li>
   <li><a href="https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/examples/src/main/java/com/google/cloud/dataflow/examples/cookbook/DatastoreWordCount.java">DatastoreWordCount</a>
   &mdash; An example that shows you how to read from Google Cloud Datastore.</li>
-  <li><a href="https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/examples/src/main/java/com/google/cloud/dataflow/examples/cookbook/DeDupExample.java">DeDupExample</a>
+  <li><a href="https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/examples/src/main/java/com/google/cloud/dataflow/examples/cookbook/DistinctExample.java">DistinctExample</a>
```

Comment:
IMO you should revert this specific change, because it makes a probably-not-dead link a dead link. In the resolution of 1078, the person handling it should be testing all links they add :)